tests/cli/post_process/test_utils.py

```diff
@@ -1,10 +1,21 @@
 import pytest
 
-from cg.cli.post_process.utils import get_post_processing_service_from_run_name
+from cg.cli.post_process.utils import (
+    UnprocessedRunInfo,
+    get_post_processing_service_from_run_name,
+    get_unprocessed_runs_info,
+)
 from cg.models.cg_config import CGConfig
 from cg.services.run_devices.pacbio.post_processing_service import PacBioPostProcessingService
 
 
+def are_lists_equal(list1: list[any], list2: list[any]) -> bool:
+    """Returns whether two lists hold the same un-hashable items, regardless of order."""
+    if len(list1) != len(list2):
+        return False
+    return all(any(obj1 == obj2 for obj2 in list2) for obj1 in list1)
```

Comment:
Does this not work using the set functionality?

Reply:
No, I could not build a set with UnprocessedRunInfo objects because they are un-hashable

Reply:
I find this way of doing it quite obscure. Could we not sort the lists and then do a direct comparison?

Reply:
Is it not sufficient to compare the run names rather than the whole objects then? I agree that this method is quite obscure and that if there is no better way to do a comparison we should think about the structure of the objects

Reply:
fixed